Rover/Log.cpp

```diff
@@ -68,13 +71,14 @@ void Rover::Log_Write_Depth()
 // @Field: Depth: Depth as detected by the sensor
 // @Field: Temp: Temperature
 
-    logger.Write("DPTH", "TimeUS,Lat,Lng,Depth,Temp",
-                        "sDUmO", "FGG00", "QLLff",
+    logger.Write("DPTH", "TimeUS,Lat,Lng,Depth,Temp, RFInstance",
+                        "sDUmO-", "FGG00-", "QLLffB",
                         AP_HAL::micros64(),
                         loc.lat,
```

Comment:
Instance, by convention and ease of reading, should be after Time, and named "Instance" and there is a unit type for instance

```suggestion
    logger.Write("DPTH", "TimeUS, Instance, Lat,Lng,Depth,Temp",
                        "s#DUmO", "F-GG00", "QBLLff",
                        AP_HAL::micros64(),
                        loc.lat,
```